jaxb/appschema/build.gradle

```diff
@@ -0,0 +1,12 @@
+// increase the version number with every schema change!
+version = '2.9.3'
```

Comment:
That was fast, I'm impressed :smile: 

Just one tiny remark - this version should represent the version of the schema that is used to build the bindings. Updates should only be needed when the schema changes / is extended.
Just wanted to make sure if this is the right version number to use. Please let me know if that number should be kept  or any other used and I will update the target platform update site so you can include the new update site URL in the hale pull request.